bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreator.java

```diff
@@ -534,8 +560,11 @@ public class ShellyThingCreator {
             Map.entry(THING_TYPE_SHELLYPLUS1PM_STR, THING_TYPE_SHELLYPLUS1PM),
             Map.entry(THING_TYPE_SHELLYPLUS1PMG3_STR, THING_TYPE_SHELLYPLUS1PM),
             Map.entry(THING_TYPE_SHELLYPLUS1PMG4_STR, THING_TYPE_SHELLYPLUS1PM),
+            Map.entry(THING_TYPE_SHELLYPLUS1L_STR, THING_TYPE_SHELLYPLUS1L),
```

Comment:
When rebasing anad resolving conflicts, use the service name string directly here:
```suggestion
            Map.entry("shellyplus1l", THING_TYPE_SHELLYPLUS1L),
```